pkgs/top-level/all-packages.nix

```diff
@@ -20322,6 +20322,10 @@ in
 
   greybird = callPackage ../data/themes/greybird { };
 
+  gruvbox-dark-gtk = callPackage ../data/themes/gruvbox-dark-gtk { };
+
+  gruvbox-dark-icons-gtk = callPackage ../data/icons/gruvbox-dark-icons-gtk { };
```

Comment:
```suggestion
  gruvbox-dark-icons-gtk = callPackage ../data/icons/gruvbox-dark-icons-gtk {
    inherit (plasma5Packages) breeze-icons;
  };
```

Reply:
I was getting an error saying `error: undefined variable 'plasma5Packages' at ...`

I tried looking around the file and found that the line other packages used was `inherit (kdeFrameworks) breeze-icons`. I tried using that instead, and it works.